esx_service/utils/auth_data.py

```diff
@@ -210,6 +210,7 @@ def get_default_datastore(self, conn):
             return str(error_info), None
         else:
             datastore_url = result[auth_data_const.COL_DEFAULT_DATASTORE_URL]
+            logging.debug("get_default_datastore: datastore_url=%s", datastore_url)
```

Comment:
nit - why this is dropped ? Looks like a useful debug info for me. 
It's a nit, feel free to merge with it